components/card/collapsible/src/index.scss

```diff
@@ -2,17 +2,15 @@
 @import '~@schibstedspain/sui-theme/lib/index';
 @import '~@schibstedspain/sui-button-basic/lib/index';
 @import '~@schibstedspain/sui-collapsible-basic/lib/index';
-
-$bgc-card-collapsible: $c-white !default;
-$h-card-collapsible-header-image: 100px;
-$c-card-collapsible-info-item: $c-gray-dark !default;
-$fz-card-collapsible-info-item-collapsed: $fz-l + 2 !default;
-$fz-card-collapsible-info-item-expanded: $fz-m !default;
-$lh-card-collapsible-info-item: 1.2 !default;
+@import 'settings';
 
 .sui-CardCollapsible {
   background-color: $bgc-card-collapsible;
 
+  &.has-shadow {
```

Comment:
What about using `is-expanded` class? so we will be able to add other properties.

Reply:
Yes, sure, I can reuse `is-expanded` class. I think it has sense since `has-shadow` is not set directly by a prop itself.